core/src/test/java/io/questdb/test/griffin/AsOfJoinTest.java

```diff
@@ -291,6 +297,80 @@ public void testExplicitTimestampIsNotNecessaryWhenSpliceJoiningExplicitlyOrdere
         testExplicitTimestampIsNotNecessaryWhenJoining("splice join", null);
     }
 
```

Comment:
Since PR affects lt and asof shouldn't there be a similar test case for asof join ?

Reply:
good one

Reply:
done